README.md

```diff
@@ -204,8 +231,66 @@ other log management system if you wish to persist logs.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
 
+#### Windows Standalone .zip
+If you don't want to use the installer script, we offer a
+.zip that can be deployed to the target host.  This bundle is available for
+download on the [Github Releases
+Page](https://github.com/signalfx/signalfx-agent/releases) for each new
+release.
+
+Before proceeding please satisfy the following three requirements are installed
+on the target host.
+* [Microsoft Visual C++ 2008 SP1 Redistributable Package (x64)](https://www.microsoft.com/en-us/download/details.aspx?id=2092)
+* [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+* [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+
+To use the bundle:
+
+1) Unzip it to a directory of your choice on the target system.
+
+2) Ensure a valid configuration file is available somewhere on the target
+system.  The main thing that the installer script provides -- but that you will
+have to provide manually with the bundle -- is a run directory for the agent to
+use.  Since you aren't installing from a package, there are three config
```

Comment:
Only two options are in the bullet points below. Is the blurb about Privileges supposed to be the third bullet?